schema/types/suricata.schema

```diff
@@ -97,17 +97,39 @@ type suricata.dcerpc = suricata.component.common + record {
   }
 }
 
+// https://github.com/OISF/suricata/blob/master/rust/src/dhcp/logger.rs
 type suricata.dhcp = suricata.component.common + record {
   dhcp: record {
-    type: string,
+    type: string, // consider switching to enum { request, reply, <unknown> }
     id: count #index=hash,
     client_mac: string #index=hash,
     assigned_ip: addr,
     client_ip: addr,
-    dhcp_type: string,
+    dhcp_type: enum {
+      discover,
+      offer,
+      request,
+      decline,
+      ack,
+      nak,
+      release,
+      inform,
+      unknown
```

Comment:
Nit: trailing comma

Reply:
I forgot to add that, can you live without it so we don't have to retrigger CI again?